app/touch-bar-menu.js

```diff
@@ -34,41 +35,65 @@ module.exports = function touchBarMenu(window, soundcloud) {
     }
   })
 
-  const trackInfo = new TouchBarLabel()
+  const repost = new TouchBarButton({
+    icon: `${__dirname}/res/repost.png`,
+    click: () => {
+      soundcloud.repost()
+    }
+  })
+
+  const titleScrubber = new TouchBarScrubber({
+    continuous: false,
+    items:[{
+      label: ''
+    }]
+  })
 
-  soundcloud.on('play', ({ title, subtitle }) => {
+  soundcloud.on('play-new-track', ({ title, subtitle, artworkURL }) => {
+    let displayTitle = `${title} by ${subtitle}`
+    displayTitle = displayTitle.padEnd(displayTitle.length * 1.3, ' ')
```

Comment:
this seems hacky but it was the only way to not have the string truncated in the touch bar. You can now swipe on the scrubber to see the whole string